lib/node.js

```diff
@@ -117,7 +117,10 @@ declare class Buffer extends Uint8Array {
 }
 
 declare module "buffer" {
+  declare var Buffer: Buffer;
```

Comment:
Taken care of. We actually need this one, mainly because having this doesn't really break anything existing because you'd have to manually import it to cause any interference. Secondly, some companies have usecases where they have to write their really focused module bundlers and those module bundlers don't add anything global like Buffer but support all requires, so they have to do `global.Buffer = require('buffer')` in their init script